static/app/components/replays/list/__stories__/replayListItem.tsx

```diff
@@ -105,12 +107,6 @@ const CardSpacing = styled('div')`
   padding: ${space(0.5)} ${space(0.5)} 0 ${space(0.5)};
 `;
 
-const ArchivedWrapper = styled(Flex)`
```

Comment:
should we make a lint rule that disallows `styled()` on our layout components like `Flex`, `Stack` etc?

it would be a good indicator to find out if we’re missing a prop, and one could always use inline-styles to work around it if really necessary